nilearn/image/tests/conftest.py

```diff
@@ -53,3 +55,28 @@ def img_4d_mni_tr2(img_4d_mni):
     header = img.header.copy()
     header["pixdim"][4] = 2.0
     return Nifti1Image(img.get_fdata(), img.affine, header=header)
+
+
+def match_headers_keys(source, target, except_keys):
```

Comment:
Is it ok to have functions defined in `conftest.py` @Remi-Gau ?

I use this in all the tests to check header fields.

Reply:
Just double checked and the pattern we have followed so far is:
- fixtures or functions used for building fixtures go into `conftest.py` files
- functions used across several tests modules either go into a `_testing.py` modules for example (`nilearn/datasets/tests/_testing.py` or `nilearn/_utils/testing.py`

So maybe just move those into a `nilearn/image/tests/_testing.py` ?

Reply:
Ok makes sense. Will do. Thanks!